### ISSUE TITLE
[BUG] : likwid-topology:56: attempt to call a nil value (field 'stdoutfalsewrite')

### ISSUE THREAD

**Issue by pramodk** [RRZE-HPC/likwid#377]
**Describe the bug**

When trying to use `likwid-topology --help` I see following:

```
$..../install/bin/likwid-topology --help
likwid-topology -- Version 5.1.0 (commit: 233ab943543480cd46058b34616c174198ba0459)
A tool to print the thread and cache topology on CPUs and GPUs.

Options:
-h, --help		 Help message
-v, --version		 Version information
-V, --verbose <level>	 Set verbosity
-c, --caches		 List cache information
-C, --clock		 Measure processor clock
-G, --gpus		 List GPU information
-O			 CSV output
-o, --output <file>	 Store output to file. (Optional: Apply text filter)
...../install/bin/likwid-lua: .../likwid/cpu/install/bin/likwid-topology:56: attempt to call a nil value (field 'stdoutfalsewrite')
stack traceback:
	.../likwid/cpu/install/bin/likwid-topology:56: in function 'usage'
	.../likwid/cpu/install/bin/likwid-topology:76: in main chunk
```

**To Reproduce**

```
wget http://ftp.fau.de/pub/likwid/likwid-5.1.0.tar.gz
tar -xzf likwid-5.1.0.tar.gz
cd likwid-5.1.0
sed -i -e "s#^PREFIX .*#PREFIX ?= $HOME/install#g" config.mk
make -j
sudo make install
```
**Additional context**

Other commands are working fine:

```
$../install/bin/likwid-topology
--------------------------------------------------------------------------------
CPU name:	Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz
CPU type:	Intel Skylake SP processor
CPU stepping:	4
********************************************************************************
Hardware Thread Topology
********************************************************************************
Sockets:		2
Cores per socket:	18
Threads per core:	2
--------------------------------------------------------------------------------
HWThread	Thread		Core		Socket		Available
0		0		0		0		*
1		0		1		0		*
2		0		2		0		*
...
```

Let me know if more information is required.

**Comment by TomTheBear** [comment]
Fixed in the `master` and the `v5.1` branch. Thanks for reporting.

**Comment by pramodk** [comment]
Ah ok! I am using last release! Thanks for confirmation!

**Comment by TomTheBear** [comment]
Probably `5.1.0`. The fix will be in the upcoming `5.1.1` release.
